packages/es-components/src/components/containers/popover/Popover.js

```diff
@@ -229,7 +233,11 @@ Popover.propTypes = {
   /** Function returning a button component to be used as the popover trigger */
   renderTrigger: PropTypes.func.isRequired,
   /** Disables popovers ability to change position to stay in viewport */
-  disableFlipping: PropTypes.bool
+  disableFlipping: PropTypes.bool,
+  /** Enable event handlers provided by Popper.js */
+  enableEvents: PropTypes.bool,
+  /** Sets the strategy for positioning the popover in Popper.js */
+  strategy: PropTypes.oneOf(['absolute', 'fixed'])
```

Comment:
Don't love the name, didn't know what it was until i read the comment. Maybe `positioningStrategy`?